### ISSUE TITLE
Button to offset tuning in all the tracks to match the tuning of your instrument

### ISSUE THREAD

**Issue by jwackito** [helge17/tuxguitar#188]
This functionality allows to play any song without changing the tuning of your instrument. It is already possible to offset the tuning but you need to do it track by track.

![image](https://github.com/helge17/tuxguitar/assets/358716/2df8c044-08be-450e-a4b1-78b882a5dde9)

This do not transpose the notes, but only the tuning of the instrument. Thus, if Guitar1 is tuned to EADGBE and offset is -2, effectively changes the Guitar1 tuning to DGCFAD. 

Love to see TuxGuitar back on development. I'm a developer myself, not Java though. In any case, if I can help, let me know.

**Comment by belfie13** [comment]
if you do all tracks then your drummer will start playing really weird stuffs 🤣🤣🤣🤣

**Comment by belfie13** [comment]
maybe transpose all pitch based instrument tracks

then you won't have your snare hits transposed into cow bell or something. anyone order blast beat hand claps or triangle?

**Comment by jwackito** [comment]
Yes, that what I meant. Transpose only the pitch based ones. In fact, I can't change the tuning of the drums tracks.
